projects/plugins/jetpack/modules/sharedaddy/admin-sharing.css

```diff
@@ -458,6 +458,9 @@ body.settings_page_sharing .advanced input[type=submit] {
 	margin-right:1rem;
 }
 
+.sharing-block-message__buttons-wrapper .button:visited {
```

Comment:
This also makes the other button's visited state #fff, which will result in:
![white letter](https://github.com/Automattic/jetpack/assets/6586048/a1de1718-62af-42eb-bb7e-eadb80a7cd93)

Reply:
Ah good spot. I've applied the style to only `button-primary`